src/server/api/controllers/users-controller.js

```diff
@@ -0,0 +1,28 @@
+const knex = require('../../config/db');
+
+const getUsers = async () => {
+  try {
+    const userList = await knex('users')
+      .select('*')
+      .limit(20);
+    return userList.length > 0 ? userList : 'No Users';
```

Comment:
So there you either return an `array of users` or a `string`. This might cause some funky errors later.

I would suggest: 
```suggestion
    return userList;
```

As then you return an array regardless and you can expect that an empty array means "No users"